### PR TITLE
Issue #72 Add console command option

### PR DESCRIPTION
**Pull request by praveenkumar**

### CODE REVIEW

**Comment by gbraad** [comment]
Error on circleci

```
cmd/crc/cmd/console.go:68: Out does not take io.Writer but has first arg os.Stdout
```

File: cmd/crc/cmd/console.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+		output.Out(result.URL)
+		return
+	}
+	output.Out("Opening the OpenShift Web console in the default browser...")
+	browser.OpenURL(result.URL)
+}
+
+func displayConsoleInMachineReadable(url string) {
+	machineDetails = fmt.Sprintf(machineDetails, url)
+	output.Out(os.Stdout, machineDetails)
```
Comment (anjannath) [inline cmd/crc/cmd/console.go]: Likely a typo here, should be `output.Out(machineDetails)`

File: pkg/crc/machine/machine.go [hunk 2]
```diff
@@ -499,3 +499,23 @@ func addNameServerToInstance(driver drivers.Driver, ns string) error {
...
+
+	if err != nil {
+		result.Success = false
+		result.Error = err.Error()
+		return *result, errors.New(err.Error())
+	}
+
+	// Here we are only checking if the VM exist and not the status of the VM
+	// We might need to improve that to console the crc status logic and only
+	// return if the Openshift is running.
```
Comment (anjannath) [inline pkg/crc/machine/machine.go]: This comment should be moved up before the `libmachine.NewClient` call

File: cmd/crc/cmd/console.go [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+	"github.com/code-ready/crc/pkg/crc/errors"
+	"github.com/code-ready/crc/pkg/crc/machine"
+	"github.com/code-ready/crc/pkg/crc/output"
+	"github.com/pkg/browser"
+	"github.com/spf13/cobra"
+)
+
+var (
+	consoleURLMode bool
+	machineDetails = `CONSOLE_URL=%s`
```
Comment (anjannath) [inline cmd/crc/cmd/console.go]: Opps, unused variable here.

**Pull request merged**